hybrid/clusterlink/none_confluent_source_cluster/README.md

```diff
@@ -0,0 +1,287 @@
+## ClusterLink Setup
+
+### Kafka Cluster with Mutual TLS Authentication
+In this example, source is running opensource kafka without any security and destination Confluent Server run in mutual TLS mode.
+
+## Set up Pre-requisites
+Set the tutorial directory for this tutorial under the directory you downloaded
+the tutorial files:
+
+```
+export TUTORIAL_HOME=$PWD/hybrid/clusterlink/none_confluent_source_cluster
+```
+
+```
+kubectl create ns destination
+```
+
+Deploy Confluent for Kubernetes (CFK) in cluster mode, so that the one CFK instance can manage Confluent deployments in multiple namespaces. Here, CFk is deployed to the `default` namespace.
+
+```
+helm upgrade --install confluent-operator \
+  confluentinc/confluent-for-kubernetes \
+  --namespace default --set namespaced=false
+```
+
+### create required secrets
+```
+kubectl -n destination create secret generic credential \
+    --from-file=plain-users.json=$TUTORIAL_HOME/creds-kafka-sasl-users.json \
+    --from-file=plain.txt=$TUTORIAL_HOME/creds-client-kafka-sasl-user.txt \
+    --from-file=basic.txt=$TUTORIAL_HOME/creds-basic-users.txt
+```
+
+### Source Cluster Deployment  
+```
```

Comment:
Are we expecting the use to know how to deploy a generic Kafka and not CP server here. Can we point to a doc to run a generic apache if available.

Reply:
@karthikeyanas , thank you for leaving the feedback, the example includes a generic AK bundled into an image the user can just deploy in the YAML, this will give them a working server to act as the source.